<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ripes::TaskTab</class>
 <widget class="QWidget" name="Ripes::TaskTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="mainLayout" stretch="3,2">
     <item>
      <layout class="QVBoxLayout" name="leftLayout" stretch="1,1,1,14,0">
       <item>
        <widget class="QLabel" name="chooseLabel">
         <property name="text">
          <string>Выбор задания</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="boxLayout" stretch="2,3">
         <item>
          <widget class="QComboBox" name="sectionBox"/>
         </item>
         <item>
          <widget class="QComboBox" name="numberBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="taskLabel">
         <property name="text">
          <string>Условия задания</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="taskText"/>
       </item>
       <item>
        <widget class="QPushButton" name="checkButton">
         <property name="text">
          <string>Проверка</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="rightLayout" stretch="0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="answerLabel">
         <property name="text">
          <string>Результаты </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="answerText">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
